src/main/java/uk/gov/gchq/gaffer/doc/function/ToTupleExample.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.doc.function;
+
+import com.google.common.collect.Lists;
+
+import uk.gov.gchq.koryphe.impl.function.ToTuple;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ToTupleExample extends FunctionExample {
+    public static void main(final String[] args) {
+        new ToTupleExample().runAndPrint();
+    }
+
+    public ToTupleExample() {
+        super(ToTuple.class);
+    }
+
+    @Override
+    protected void runExamples() {
+        toTuple();
+    }
+
+    private void toTuple() {
+        // ---------------------------------------------------------
+        final ToTuple function = new ToTuple();
+        // ---------------------------------------------------------
+
+        Map<String, Object> input = new HashMap<>();
+        input.put("A", 1);
```

Comment:
This is fine except some of the inputs look a bit strange:

Input Type | Input | Result Type | Result
-- | -- | -- | --
java.util.ArrayList | [1, 2, 3, 4] | [java.lang.Integer, java.lang.Integer, java.lang.Integer, java.lang.Integer] | [1, 2, 3, 4]
[I | [I@6ae448f6 | [java.lang.Integer, java.lang.Integer, java.lang.Integer, java.lang.Integer] | [1, 2, 3, 4]
[Ljava.lang.Integer; | [1, 2, 3, 4] | [java.lang.Integer, java.lang.Integer, java.lang.Integer, java.lang.Integer] | [1, 2, 3, 4]
java.util.HashMap | {A=1, B=2, C=3} | uk.gov.gchq.koryphe.tuple.MapTuple | [1, 2, 3]